src/main/java/socialite/logic/commands/FindCommand.java

```diff
@@ -14,14 +14,17 @@
 
     public static final String COMMAND_WORD = "find";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
-            + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Filters contacts based on specified keywords.\n"
+            + "Keywords can be in the form of tags (prefix with 't/'),\n"
```

Comment:
Might consider removing some of the linebreaks here to standardise with other types of commands.